packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6555,4 +6555,22 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return Promise.reject(new Error('Error sending reaction, service url not found.'));
   }
+
+  /**
+   * Method to enable or disable reactions inside the meeting.
+   *
+   * @param  {boolean} enable - enable or disable reactions
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
+   */
+  public toggleReactions(enable: boolean) {
+    const isEnabled = this.locusInfo?.controls?.reactions?.enabled;
```

Comment:
one change though, since we are using safe operator, do you want to add tests when its not enabled as the object is missing?

Reply:
added